index.js

```diff
@@ -3,30 +3,84 @@
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @returns {boolean} Returns `true` if all values in the array are numbers. Otherwise, return `false`.
  */
-function isValid(rolls) {}
+function isValid(rolls) {
+
+let flag = true
+if(rolls.length === 0) {
+  return true
+}
+  for (let i = 0; i < rolls.length; i++) {
+    if(typeof rolls[i] === 'number') {
+      return flag
+    } 
+    flag = false
+    return flag
+  }
+  
+}
+
+//return rolls.every(element => typeof element === 'number')
 
 /**
  * Finds a value in an array. If that value is in the array, returns it. Otherwise, returns `null`.
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @param {number} value - A specific value to find.
  * @returns {*} - The found value or `null`.
  */
-function findValue(rolls, value) {}
+function findValue(rolls, value) {
+  
+ if(rolls.length ===0) {
+  return null
+ } if(rolls.includes(value)) {
+  return value
+}   else {
+  return null
+}
+    }
+    
+//return (!rolls.length || !rolls.includes(value) ? null : value)
 
 /**
  * Returns a new array from the `rolls` array with only values equal to or greater than the `lowest` value.
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @param {number} lowest - A number that represents the lowest allowed value in the new array.
  * @returns {number[]} An array of all numbers that are equal to or higher than the `lowest` value.
  */
-function filterOutLowValues(rolls, lowest) {}
+function filterOutLowValues(rolls, lowest) {
+  let greater = []
+  if(rolls.length === 0) {
```

Comment:
this conditional is unnecessary. If `rolls` is an empty array it will not run the for loop and will return on line 62